pages/linux/at.md

```diff
@@ -0,0 +1,15 @@
+# at
+
+> Executes commands at a specified time.
+
+- Open `at` prompt to execute commands at specific time. When you're done, press Ctrl+D to save the schedule:
+
+`at {{hh:mm:ss}}`
+
+- Execute commands and email result:
```

Comment:
Iw we want to shorten the description, I'm wondering if we should cut out the expanded acronym instead, since I suspect more people will have heard of sendmail than of "Mail Transfer Agent". I.e. "using a local MTA such as sendmail". I'm not 100% sure this is better though.... perhaps drop the MTA altogether and say simply "using a local mailing program such as sendmail"?